responses.py

```diff
@@ -534,18 +534,38 @@ def replace(self, method_or_response=None, url=None, body="", *args, **kwargs):
         self._matches[index] = response
 
     def add_callback(
-        self, method, url, callback, match_querystring=False, content_type="text/plain"
+        self,
+        method,
+        url,
+        callback,
+        match_querystring=False,
+        stream=False,
```

Comment:
This parameter should come after `content_type`. People *could* be using positional arguments for `content_type` and adding a new parameter in the middle will break that use-case.

Reply:
Thanks, I didn't consider that; I'll change that back